tests/unit/ResultsItemDetails.spec.js

```diff
@@ -0,0 +1,155 @@
+    }
+
+    store = new Vuex.Store({
+      state,
+      mutations
+    })
+
+    computed = {
+      park() { 
+        return { 
+          formatted_address: "5123-5275 Valmont Rd, Boulder",
+          geometry: {},
+          name: "BoneYard",
+          opening_hours: {open_now: false},
+          photos: [{}],
+          rating:4.7        
+        }
+      },
+      open() {
+        return 'Closed Now'
+      },
+      saved() {
+        return 'SAVE'
+      }
+    },
+    $route = {
+      path: '/results/BoneYard',
+      params: {
+        name: 'BoneYard'
+      }
+    }
+  })
+
+  it('should render the component', () => {
+    const wrapper = shallowMount(ResultsItemDetails, { 
+      store, 
+      localVue, 
+      computed,
+      mocks: {
+        $route
+      },
+      data() {
+        return {
+          parkName: 'BoneYard'
+        }
+      }
+    })
+
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.find('section').isVisible()).toBeTruthy();
+    expect(wrapper.find('h1').text()).toBe('BoneYard');
+    expect(wrapper.find('h2').text()).toBe('Not the right park for your pup?');
+
+    const getAllPTags = wrapper.findAll('p');
+    expect(getAllPTags.at(0).text()).toBe('Address: 5123-5275 Valmont Rd, Boulder');
+    expect(getAllPTags.at(1).text()).toBe('Closed Now');
+    expect(getAllPTags.at(2).text()).toBe('Rating: 4.7 / 5');
+
+    const getAllButtonTags = wrapper.findAll('button');
+    expect(getAllButtonTags.at(0).text()).toBe('SAVE');
+    expect(getAllButtonTags.at(1).text()).toBe('Get Directions');
+    expect(getAllButtonTags.at(2).text()).toBe('Search Again');
+  })
+
+  it('should trigger savePark method when save button is clicked', () => {
+    const wrapper = shallowMount(ResultsItemDetails, { 
+      store, 
+      localVue, 
+      computed,
+      mocks: {
+        $route
+      },
+      data() {
+        return {
+          parkName: 'BoneYard'
+        }
+      }
+    }) 
+
+    wrapper.setMethods({ savePark: jest.fn() });
+    expect(wrapper.find('.button-save-park').exists()).toBe(true);
+    const saveParkButton = wrapper.find('.button-save-park');
+    saveParkButton.trigger('click');
+    expect(wrapper.vm.savePark).toHaveBeenCalledTimes(1);
+  })
+
+  it.skip('should trigger the savePark mutation', () => {
+    const wrapper = shallowMount(ResultsItemDetails, { 
+      store, 
+      localVue, 
+      computed,
+      mocks: {
+        $route
+      },
+      data() {
+        return {
+          parkName: 'BoneYard'
+        }
+      }
+    }) 
+    const saveParkButton = wrapper.find('.button-save-park');
+    saveParkButton.trigger('click');
+    expect(store._mutations.savePark).toHaveBeenCalled();
+    //This test is currently failing.
+    //I believe this is due to the fact that the savePark method
+    //is not being mocked directly.  I am wondering if it is not
+    //possible to test the mutation this way, and that it needs to 
+    //be an isolated test...
+  })
```

Comment:
Maybe you've already see this, but possibly some good info here: https://vue-test-utils.vuejs.org/v2/guide/vuex.html#testing-with-a-mock-store